src/SDKs/ContainerRegistry/dataplane/Microsoft.Azure.ContainerRegistry/Microsoft.Azure.ContainerRegistry.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.ContainerRegistry</PackageId>
+    <Description>Provides management capabilities for Azure Container Registry.</Description>
+    <AssemblyTitle>Microsoft Azure Container Registry</AssemblyTitle>
+    <Version>1.0.0</Version>
```

Comment:
Since the REST API version is in preview, the Nuget package must also be preview, versioning for preview must start with `0.9.0-preview`